diff --git a/js/bootstrap/collapse.js b/js/bootstrap/collapse.js
index be27c9e..acc2560 100644
--- a/js/bootstrap/collapse.js
+++ b/js/bootstrap/collapse.js
@@ -175,7 +175,7 @@
       var data    = $this.data('bs.collapse')
       var options = $.extend({}, Collapse.DEFAULTS, $this.data(), typeof option == 'object' && option)

-      if (!data && options.toggle && /show|hide/.test(option)) options.toggle = false
+      if (!data && options.toggle && (typeof option === 'string') && /show|hide/.test(option)) options.toggle = false
       if (!data) $this.data('bs.collapse', (data = new Collapse(this, options)))
       if (typeof option == 'string') data[option]()
     })
